frontend/src/pages/Result.jsx

```
@@ -0,0 +1,106 @@
+              onClick={fetchResults}
+              className="w-full py-2 text-white bg-blue-500 rounded-md hover:bg-blue-600 focus:outline-none focus:ring-2 focus:ring-blue-500"
+            >
+              Fetch Results
+            </button>
+          </div>
+        </div>
+
+        {/* Loading and Error Messages */}
+        {loading && (
+          <div className="text-xl text-center text-gray-600">
+            <div className="w-10 h-10 mx-auto mb-4 border-4 border-t-4 border-blue-500 rounded-full animate-spin"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)